script/libtest.bash

```diff
@@ -15,7 +15,7 @@
 # k8s and kubeless helpers, specially "wait"-ers on pod ready/deleted/etc
 
 KUBELESS_JSONNET=kubeless.jsonnet
-KUBELESS_JSONNET_RBAC=kubeless-rbac-novols.jsonnet
```

Comment:
is the `novols` file used anymore? If not you can delete it

Reply:
Agreed. It's not useful anymore.